### PR TITLE
NP-2474: update libraries to use central dependencies file

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,30 +1,10 @@
...
+    id 'nva.identity.service.rootplugin'
 }
 
-allprojects {
+//workaround for jacoco-merge to work
+allprojects{
     apply plugin: 'nva.identity.service.java-common-conventions'
 }
 
-subprojects {
```
Comment (axthosarouris) [inline build.gradle:9]: moved to dynamodbloca plugin

File: buildSrc/src/main/groovy/nva.identity.service.dynamodblocal.gradle [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+
```
Comment (axthosarouris) [inline buildSrc/src/main/groovy/nva.identity.service.dynamodblocal.gradle:2]: moved from main build.gradle file

File: buildSrc/src/main/groovy/nva.identity.service.jacoco-merge.gradle [hunk 3]
```diff
@@ -0,0 +1,66 @@
+//For generating one unified coverage report
```
Comment (axthosarouris) [inline buildSrc/src/main/groovy/nva.identity.service.jacoco-merge.gradle:1]: the jacoco-merge should be applied only on the top project. Therefore I put the logic in a single plugin and used the plugin in the main build.gradle file

File: buildSrc/src/main/groovy/nva.identity.service.gradlelint.gradle [hunk 4]
```diff
@@ -0,0 +1,7 @@
+plugins{
```
Comment (axthosarouris) [inline buildSrc/src/main/groovy/nva.identity.service.gradlelint.gradle:1]: The nebula-lint plugin can only be applied on the root project. Therefore I created a plugin to be applied to the root project

File: user-access-handlers/src/test/java/no/unit/nva/handlers/GetRoleHandlerTest.java [hunk 5]
```diff
@@ -125,7 +125,7 @@ private ByteArrayOutputStream sendGetRoleRequest() throws IOException {
         RequestInfo requestInfo = queryWithRoleName(THE_ROLE);
         InputStream requestStream = new HandlerRequestBuilder<>(objectMapper)
             .withPathParameters(requestInfo.getPathParameters())
-            .withHttpMethod(HttpMethod.GET.asString())
+            .withHttpMethod(HttpMethod.GET.toString())
```
Comment (axthosarouris) [inline user-access-handlers/src/test/java/no/unit/nva/handlers/GetRoleHandlerTest.java:128]: compilation error after library upgrade

File: user-access-public-model/src/test/java/no/unit/nva/useraccessmanagement/model/RoleDtoTest.java [hunk 6]
```diff
@@ -118,7 +118,7 @@ public void userDtoCannotBeCreatedWithoutTypeValue() throws InvalidEntryInternal
 
         Executable action = () -> objectMapper.readValue(jsonStringWithoutType, RoleDto.class);
         InvalidTypeIdException exception = assertThrows(InvalidTypeIdException.class, action);
-        assertThat(exception.getMessage(), StringContains.containsString(INVALID_TYPE_EXCEPTION_MESSAGE_SAMPLE));
+        assertThat(exception.getMessage(), StringContains.containsString(RoleDto.TYPE));
```
Comment (axthosarouris) [inline user-access-public-model/src/test/java/no/unit/nva/useraccessmanagement/model/RoleDtoTest.java:121]: message has changed in the latest nva-commons

**Review by brinxmat — APPROVED** [review]

**Pull request merged**